src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultDisplayTemplates.cs

```diff
@@ -237,23 +234,16 @@ public static IHtmlContent ObjectTemplate(IHtmlHelper htmlHelper)
                     continue;
                 }
 
-                var divTag = new TagBuilder("div");
-
                 if (!propertyMetadata.HideSurroundingHtml)
                 {
                     var label = propertyMetadata.GetDisplayName();
                     if (!string.IsNullOrEmpty(label))
                     {
-                        divTag.SetInnerText(label);
-                        divTag.AddCssClass("display-label");
-                        content.AppendLine(divTag.ToHtmlContent(TagRenderMode.Normal));
-
-                        // Reset divTag for reuse.
-                        divTag.Attributes.Clear();
-                    }
-
-                    divTag.AddCssClass("display-field");
-                    content.Append(divTag.ToHtmlContent(TagRenderMode.StartTag));
+                        var labelTag = new TagBuilder("div");
```

Comment:
Well, most `TagBuilder` variables in product code are named `tagBuilder` which truly conveys the _type_. Even that works better here than `labelTag` because the semantics of the generated `<div>` are not the same as the contained `<label>` element.

BTW we also name `TagBuilder` variables after the generated element many other places, starting probably in MVC 1.0. The reasoning might be along the lines "The _semantics_ of this `TagBuilder` are that it will generate a `<div>` element." Are you suggesting we make a general change throughout MVC?